src/aspire/source/coordinates.py

```diff
@@ -339,8 +339,8 @@ def import_relion_ctf(self, ctf):
         """
         data_block = RelionStarFile(ctf).get_merged_data_block()
 
-        # data_block is a pandas Dataframe containing the micrographs
-        if not len(data_block) == self.num_micrographs:
+        # data_block is a dict containing the micrographs
+        if not len(data_block[list(data_block.keys())[0]]) == self.num_micrographs:
```

Comment:
Cleaner way to do this? `list(data_block.values())[0]`? Maybe just as ugly…

Reply:
Yeah, surprised I missed that one. Changed.